js/packages/openinference-core/src/trace/promptTemplate.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  PROMPT_TEMPLATE_TEMPLATE,
+  PROMPT_TEMPLATE_VARIABLES,
+  PROMPT_TEMPLATE_VERSION,
+} from "@arizeai/openinference-semantic-conventions";
+import { Attributes, Context, createContextKey } from "@opentelemetry/api";
+
+export const PROMPT_TEMPLATE_TEMPLATE_KEY = createContextKey(
+  `OpenInference SDK Context Key ${PROMPT_TEMPLATE_TEMPLATE}`,
+);
+
+export const PROMPT_TEMPLATE_VARIABLES_KEY = createContextKey(
+  `OpenInference SDK Context Key ${PROMPT_TEMPLATE_VARIABLES}`,
+);
+
+export const PROMPT_TEMPLATE_VERSION_KEY = createContextKey(
+  `OpenInference SDK Context Key ${PROMPT_TEMPLATE_VERSION}`,
+);
+
+export function setPromptTemplateAttributes({
+  context,
+  template,
+  variables,
+  version,
+}: {
+  context: Context;
```

Comment:
I like this structure but if we do it we should probably make it consistent across all